lib/authenforce/enforcers.go

```diff
@@ -1,31 +1,68 @@
 package authenforce
 
 import (
+	"context"
+	"sync/atomic"
+
 	"github.com/cuvva/cuvva-public-go/lib/cher"
-	"github.com/wearemojo/mojo-public-go/lib/authparsing"
+	"github.com/wearemojo/mojo-public-go/lib/errgroup"
+	"github.com/wearemojo/mojo-public-go/lib/merr"
+)
+
+type (
+	Enforcers   []Enforcer
```

Comment:
Do we want a function for creating this - eg wrap

also should prob move the info on misuse here